spinnaker_camera_driver/CMakeLists.txt

```diff
@@ -108,10 +107,6 @@ if(EXISTS ${CMAKE_CURRENT_BINARY_DIR}/usr/lib)
            PATTERN libQt* EXCLUDE)
 else()
   get_filename_component(Spinnaker_LIB_DIR ${Spinnaker_LIBRARIES} DIRECTORY)
-  file(READ ${PROJECT_SOURCE_DIR}/cmake/install_files filenames)
```

Comment:
can you explain the issue behind this removal?
just want to be careful about changes in case we end up with something to contribute upstream.

Reply:
it was breaking on `libSpinnaker.so.1` in `install_files` because we have `libSpinnaker.so.2` installed. We install spinnaker outside of the build process of this package, so it didn't seem right to try to install it again here.